frontend/src/Components/loginForm.tsx

```diff
@@ -0,0 +1,55 @@
+import React, { useState } from 'react';
+import axios from 'axios';
+import './login.css';
+
+export const LoginForm = () => {
+    const [username, setUsername] = useState<string>('');
+    const [password, setPassword] = useState<string>('');
+    
+    const handleSubmit = async (event: React.FormEvent<HTMLFormElement>) => {
+        event.preventDefault();
+        try {
+          const response = await axios.post("http://localhost:8000/login/password", {
+            username,
+            password,
+          });
+          console.log(response.data)
+        } catch(error) {
+            console.error(error);
+        }
+    };
+    
+  const handleUsernameChange = (event: React.ChangeEvent<HTMLInputElement>) => {
+    setUsername(event.target.value);
+  };
+
+  const handlePasswordChange = (event: React.ChangeEvent<HTMLInputElement>) => {
+    setPassword(event.target.value);
+  };
+
+    return (
+        <section>
+          {/* TODO (Fahad) T78: refactor this code after wewrap logo can be downloaded */}
+          <img src='./logo192.png' alt='placeholder for wewrap logo'></img>
+          <h1>Log in to your account</h1>
+          <p className='subheader'>Don't have an account? Sign up</p>
+
+          <form action="/login/password" method="post" onSubmit={handleSubmit}>
+              <div>
+                  <p className='form_text'>username*</p>
+                  <label htmlFor="username"></label>
```

Comment:
We should use the `label` to provide the actual label for the input rather than a `p` tag.

Reply:
https://stackoverflow.com/questions/11197671/use-css-to-automatically-add-required-field-asterisk-to-form-inputs

also a cool way to do this as well. You'll want to also probably create some CSS to make the label format the way you want it.

Reply:
Fixed!